<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedialibDlg</class>
 <widget class="QDialog" name="MedialibDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>媒体库</string>
  </property>
  <widget class="CButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnReturn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>48</y>
     <width>420</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="CButton" name="btnUpward">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnUpward.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/btnPlay.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameFilterLanguage">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>585</width>
     <height>60</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="CLabel" name="labelDemandCN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>国语</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelDemandHK">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>粤语</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelDemandKR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>韩语</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelDemandEN">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>英文</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelDemandEUR">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>小语种</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>labelDemandHK</zorder>
   <zorder>labelDemandKR</zorder>
   <zorder>labelDemandEN</zorder>
   <zorder>labelDemandEUR</zorder>
   <zorder>labelDemandCN</zorder>
  </widget>
  <widget class="CLabel" name="labelSingerImg">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>40</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>labelTitle</zorder>
  <zorder>labelSingerImg</zorder>
  <zorder>btnReturn</zorder>
  <zorder>btnUpward</zorder>
  <zorder>btnPlay</zorder>
  <zorder>frameFilterLanguage</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CButton</class>
   <extends>QPushButton</extends>
   <header location="global">widget/widget/button.h</header>
  </customwidget>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header location="global">widget/widget/label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
